### PR TITLE
SOV-4012: bob staking history

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-4012

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx [hunk 1]
```diff
@@ -51,6 +52,7 @@ export const COLUMNS_CONFIG = [
       <TransactionIdRenderer
         hash={item.id.split('-')[0]}
         dataAttribute="staking-delegate-history-tx-hash"
+        chainId={getCurrentChain()}
```
Comment (rick23p) [inline apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx:55]: shouldn't it be the value from `useCurrentChain()`?

File: apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingExtendedDuration/StakingExtendedDuration.constants.tsx [hunk 2]
```diff
@@ -54,6 +55,7 @@ export const COLUMNS_CONFIG = [
       <TransactionIdRenderer
         hash={item.id.split('-')[0]}
         dataAttribute="staking-extend-history-tx-hash"
+        chainId={getCurrentChain()}
```
Comment (rick23p) [inline apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingExtendedDuration/StakingExtendedDuration.constants.tsx:58]: the same thing here

File: apps/frontend/src/constants/general.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export const SUBGRAPH = {
   ZERO: process.env.REACT_APP_GRAPH_ZERO,
   MYNT: process.env.REACT_APP_GRAPH_MYNT,
   BOB: process.env.REACT_APP_GRAPH_BOB,
+  BOB_SUBGRAPH: process.env.REACT_APP_GRAPH_BOB_STAKING,
```
Comment (rick23p) [inline apps/frontend/src/constants/general.ts]: `BOB_STAKING` ?

**Review by rick23p — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx (hunk 1 above)
Comment (creed-victor) [reply apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx:55]: It returns same thing, just that `getCurrentChain()` is not reactive. Can be used inline so don't need to have extra variable.

**Review by rick23p — APPROVED** [review]

**Pull request merged**